includes/cf-descriptions/dts-x.md

```diff
@@ -0,0 +1,5 @@
+**DTS-X**<br>
+
+[DTS:X is an object-based audio codec](https://www.whathifi.com/advice/dtsx-what-it-how-can-you-get-it), which aims to create a multi-dimensional sound that “moves around you like it would in real life”.
+You may think that sounds a lot like Dolby Atmos, and you’d be right. But where DTS:X differs lies in the required speaker configuration. While Dolby Atmos requires you to add extra overhead channels to your 5.1 or 7.1 setup, DTS:X works with standard surround speaker setups – just like the one you might already have at home. It can support up to 32 speaker locations and up to an 11.2-channel system.
+DTS:X like other higher quality DTS formats (e.g. DTS-HD MA) is lossless.
```

Comment:
```suggestion
[DTS:X is an object-based audio codec](https://www.whathifi.com/advice/dtsx-what-it-how-can-you-get-it) which aims to create a multi-dimensional sound that “moves around you like it would in real life”.
You may think that sounds similar to Dolby Atmos, and you’d be right, but where DTS:X's differences lie in the required speaker configuration. While Dolby Atmos requires adding additional overhead channels to your 5.1 or 7.1 setup, DTS:X works with standard surround speaker setups – just like the one you might already have at home. It can support up to 32 speaker locations and an 11.2-channel system.
DTS:X, like other higher-quality DTS formats (e.g. DTS-HD MA), is lossless.